test/integration/test.speech_to_text.js

```diff
@@ -380,5 +380,42 @@ describe('speech_to_text_integration', function() {
       speech_to_text.deleteCustomization({ customization_id: customization_id }, done);
       customization_id = null;
     });
+
+    describe('asynchronous api', function() {
+      let jobId = null;
+
+      it('registerCallback()', function(done) {
+        const params = { callback_url: 'http://watson-test-resources.mybluemix.net/results', user_secret: 'ThisIsMySecret' };
+        speech_to_text.registerCallback(params, done);
+      });
+
+      it('createRecognitionJob()', function(done) {
+        const params = {
+          audio: fs.createReadStream(__dirname + '/../resources/weather.wav'),
+          content_type: 'audio/l16;rate=41100',
+          callback_url: 'http://watson-test-resources.mybluemix.net/results',
+          user_token: 'ThisIsMySecret',
+          event: 'recognitions.completed',
+          results_ttl: 60
+        };
+        speech_to_text.createRecognitionJob(params, function(err, res) {
+          assert.ifError(err);
+          jobId = res.id;
+          done();
+        });
+      });
+
+      it('getRecognitionJobs()', function(done) {
+        speech_to_text.getRecognitionJobs(done);
+      });
+
+      it('getRecognitionJob()', function(done) {
+        speech_to_text.getRecognitionJob({ id: jobId }, done);
+      });
+
+      it('deleteRecognitionJob()', function(done) {
+        speech_to_text.deleteRecognitionJob({ id: jobId }, done);
```

Comment:
Do you have a set of credentials, and does this test pass when you run it locally?

It looks for credentials in `test/resources/auth.js`, and if that file doesn't exist, then it skips all integration tests.

You can add them for just this service by creating the file with content like so:

```js
'use strict';

module.exports = {
  speech_to_text: {
    url: 'https://stream.watsonplatform.net/speech-to-text/api',
    username: '<your username>',
    password: '<your password>',
    headers: {
      'X-Watson-Learning-Opt-Out': 1,
      'X-Watson-Test': 1
    },
    version: 'v1'
  },
  // other services would go here
};
```

With that in place, you can run

    node ./node_modules/mocha/bin/_mocha /Users/nfriedly/node-sdk/test/integration/test.speech_to_text.js --grep "speech_to_text_integration asynchronous api "

(Note that this command expects you have already made the above change to move it outside of the customization describe.)

Note that with that file in place, the `npm test` command will try to run *all* integration tests, and the rest of them will fail due to missing credentials.  I'm going to fix that eventually, but in the meanwhile, you can just rename it to something else.